core/templates/dev/head/components/search_bar_directive.html

```diff
@@ -27,7 +28,7 @@
           ng-if="selectionDetails.categories.selections[item.id]">
         <a href ng-click="toggleSelection('categories', item.id)">
           <[item.text]>
-          <span ng-if="selectionDetails.categories.selections[item.id]" class="glyphicon glyphicon-ok pull-right"></span>
+          <i ng-if="selectionDetails.categories.selections[item.id]" class="material-icons pull-right">&#xE876;</i>
```

Comment:
These look a little large. Can they be smaller -- maybe md-18? Ditto throughout this file.